tests/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ if (NOT ${LIBDOMAIN_BUILD_TESTS})
   return()
 endif()
 
+include(CTest)
+
```

Comment:
We should include `ctest` in the `CMakeLists.txt` file located at the project's root directory.